sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml

```diff
@@ -448,6 +475,10 @@
                       <pattern>com.google</pattern>
                       <shadedPattern>${shadingPrefix}.com.google</shadedPattern>
                     </relocation>
+                    <relocation>
+                      <pattern>com.azure.resourcemanager</pattern>
+                      <shadedPattern>${shadingPrefix}.com.azure.resourcemanager</shadedPattern>
+                    </relocation>
                   </relocations>
```

Comment:
Not from this change but there is a duplicated entry for `io.netty` in shaded jars. Could remove one along this change.

Reply:
Done

Reply:
@stevenylai  - hmm, I did not see the duplicate io.netty entry, can you elaborate a little bit more about it? how you find it etc? And feel free to create a github issue for tracking this issue, thanks :)